### DISCUSSION TITLE
Error trying to connect: tcp connect error: Connection refused (os error 111) ?

### DISCUSSION THREAD

**Discussion by 0xconstructor** [foundry-rs/foundry#5127] [Help]
any one help me to solve this issue regarding the TCP connection establishment ,
![Screenshot (38)](https://github.com/foundry-rs/foundry/assets/123366816/e55a4a39-6a61-4783-9e3c-5e3523f5891e)

**Comment by Niferu** [comment]
Hi @web3baji 

You are probably just listening to wrong url. Once you run `anvil` check below:

![image](https://github.com/foundry-rs/foundry/assets/110537615/2117b464-4448-4000-8c75-ffd35bdab619)

then in your command use proper url
